src/mime.cc

```diff
@@ -262,70 +259,73 @@ mimeInit(char *filename)
-        }
+            xstrncpy(chopbuf, buf, BUFSIZ);
 
-        if ((icon = strtok(nullptr, w_space)) == nullptr) {
-            debugs(25, DBG_IMPORTANT, "ERROR: mimeInit: parse failure: '" << buf << "'");
-            continue;
-        }
+            const auto pattern = SBuf(strtok(chopbuf, w_space));
+            if (pattern.isEmpty()) {
+                debugs(25, DBG_IMPORTANT, "ERROR: mimeInit: parse failure: '" << buf << "'");
+                continue;
+            }
 
-        if ((encoding = strtok(nullptr, w_space)) == nullptr) {
-            debugs(25, DBG_IMPORTANT, "ERROR: mimeInit: parse failure: '" << buf << "'");
-            continue;
-        }
+            if ((type = strtok(nullptr, w_space)) == nullptr) {
+                debugs(25, DBG_IMPORTANT, "ERROR: mimeInit: parse failure: '" << buf << "'");
+                continue;
+            }
 
-        if ((mode = strtok(nullptr, w_space)) == nullptr) {
-            debugs(25, DBG_IMPORTANT, "ERROR: mimeInit: parse failure: '" << buf << "'");
-            continue;
-        }
+            if ((icon = strtok(nullptr, w_space)) == nullptr) {
+                debugs(25, DBG_IMPORTANT, "ERROR: mimeInit: parse failure: '" << buf << "'");
+                continue;
+            }
 
-        download_option = 0;
-        view_option = 0;
+            if ((encoding = strtok(nullptr, w_space)) == nullptr) {
+                debugs(25, DBG_IMPORTANT, "ERROR: mimeInit: parse failure: '" << buf << "'");
+                continue;
+            }
 
-        while ((option = strtok(nullptr, w_space)) != nullptr) {
-            if (!strcmp(option, "+download"))
-                download_option = 1;
-            else if (!strcmp(option, "+view"))
-                view_option = 1;
-            else
-                debugs(25, DBG_IMPORTANT, "ERROR: mimeInit: unknown option: '" << buf << "' (" << option << ")");
-        }
+            if ((mode = strtok(nullptr, w_space)) == nullptr) {
+                debugs(25, DBG_IMPORTANT, "ERROR: mimeInit: parse failure: '" << buf << "'");
+                continue;
+            }
 
-        if (regcomp(&re, pattern, re_flags) != 0) {
-            debugs(25, DBG_IMPORTANT, "ERROR: mimeInit: regcomp failure: '" << buf << "'");
-            continue;
-        }
+            download_option = 0;
+            view_option = 0;
+
+            while ((option = strtok(nullptr, w_space)) != nullptr) {
+                if (!strcmp(option, "+download"))
+                    download_option = 1;
+                else if (!strcmp(option, "+view"))
+                    view_option = 1;
+                else
+                    debugs(25, DBG_IMPORTANT, "ERROR: mimeInit: unknown option: '" << buf << "' (" << option << ")");
+            }
 
-        m = new MimeEntry(pattern,re,type,encoding,mode,view_option,
-                          download_option,icon);
+            m = new MimeEntry(SBuf(pattern),type,encoding,mode,view_option,download_option,icon);
```

Comment:
Isn't pattern already a SBuf?

Reply:
@yadij, please note that this PR was cleared for merging without addressing Francesco's and [my](https://github.com/squid-cache/squid/pull/1233/files?w=0#r1064229147) change requests about `pattern` type (those two change requests are related/linked). Those change requests may be completely wrong, but clearing a PR without changing code or even commenting on them increases the pressure on the reviewers to _block_ future PRs (to make sure their minor polishing requests are not similarly seemingly ignored), increasing overheads for everybody.